src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,17 +1,43 @@
 #!/usr/bin/python
 import sys
 import swsssdk
+import redis
+import time
+import syslog
 
 argc = len(sys.argv)
-if argc == 2 and sys.argv[1] == '-h':
-    print("""
+if argc == 2:
+    op = sys.argv[1]
+    if op == '-h':
+        print("""
 Example 1: sonic-db-cli CONFIG_DB keys *
 Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
 Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
 Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+Example 4: sonic-db-cli PING
```

Comment:
>PING [](start = 24, length = 4)

What if there is a dbName called PING/SAVE/FLUSHALL in database_config.json?

Reply:
if dbName is PING/SAVE/FLUSHALL, then following cmd args are expected. If there is no cmd args, then the only args is PING/SAVE/FLUSHALL and it is the unary operation we want.

Other case, print helper string